common.js

```diff
@@ -0,0 +1,35 @@
+/* Build DID Method table using JSON data */
+async function buildDidMethodTables(config) {
+  const {document} = window;
+  const response = await fetch('methods/index.json');
+  const allMethods = await response.json();
+  console.log(allMethods);
```

Comment:
please comment or remove console log.

Reply:
Agree, errant debugging statement that was not meant to be printed to the console. Will fix.

Reply:
Fixed in latest.